controllers/grid/users/reviewer/form/ThankReviewerForm.php

```diff
@@ -149,13 +149,16 @@ public function execute(...$functionArgs)
         // update the ReviewAssignment with the acknowledged date
         $newData = ['dateAcknowledged' => Core::getCurrentDate()];
         if (!in_array($reviewAssignment->getConsidered(), [ReviewAssignment::REVIEW_ASSIGNMENT_CONSIDERED, ReviewAssignment::REVIEW_ASSIGNMENT_RECONSIDERED])) {
```

Comment:
Not sure when this could happen... because in UI its needs to be confirmed, before thanking.  But since its here I followed same logic for the NotifyReviewer